### PR TITLE
Added Json for Camel Connectors Grafana dashboard

### PR DESCRIPTION
**Pull request by shivamG640**
Hi @lgarciaaco @heiko-braun @abkieling
Kindly review!
Use the following link to see the dashboard created with this commit:
https://grafana-route-redhat-openshift-connectors-observability.apps.stage.etcj.s1.devshift.org/d/nnrd9pu7k/connectors?var-namespace=redhat-openshift-connectors-c95j6h66ar2c9mtr5vjg&var-pod=mctr-c9c380sb6db8jvtshdog-f7cb77cc4-tsd7z

### CODE REVIEW

**Comment by shivamG640** [comment]
Hi @lgarciaaco ,
The modified dashboard can be seen using the following link:
https://grafana-route-redhat-openshift-connectors-observability.apps.stage.etcj.s1.devshift.org/d/nnrd9pu7k/connectors?orgId=1&from=now-1h&to=now&refresh=30s&var-namespace=redhat-openshift-connectors-c95j6h66ar2c9mtr5vjg&var-pod=mctr-ca5osef46g0ae32av9k0-7b6c67964-nz6qd

**Comment by lgarciaaco** [comment]
@shivamG640 can you open this PR agains development branch first? That way we can test in rhoc-dev cluster first

**Comment by shivamG640** [comment]
@lgarciaaco I have changed the target branch to development.

File: observability/resources/grafana/camelk-connectors.yaml [hunk 1]
```diff
@@ -0,0 +1,1431 @@
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  labels:
+    app: rhoc
+  name: camelk-operators
```
Comment (lgarciaaco) [inline observability/resources/grafana/camelk-connectors.yaml:6]: maybe the name should be connector related and not operator

File: observability/resources/grafana/camelk-connectors.yaml [hunk 2]
```diff
@@ -0,0 +1,1431 @@
...
+spec:
+  name: camelk-operators.json
```
Comment (lgarciaaco) [inline observability/resources/grafana/camelk-connectors.yaml:8]: maybe the name should be connector related and not operator

**Review by lgarciaaco — APPROVED** [review]

**Pull request merged**